packages/mainsail/source/fee.service.test.ts

```diff
@@ -0,0 +1,133 @@
+            assert.snapshot("fees_mainsail_username_registration", result.usernameRegistration);
+            assert.snapshot("fees_mainsail_username_resignation", result.usernameResignation);
+            assert.snapshot("fees_mainsail_vote", result.vote);
+        });
+    });
+
+    describe("calculate", () => {
+        let feeService: FeeService;
+        let transactionService: TransactionService;
+        let defaultTransferInput: any;
+
+        beforeAll(async () => {
+            feeService = await createService(FeeService, "mainsail.devnet");
+            transactionService = await createService(TransactionService, "mainsail.devnet", (container) => {
+                container.constant(IoC.BindingType.Container, container);
+                container.factory(MultiSignatureSigner);
+                container.constant(IoC.BindingType.DataTransferObjects, {
+                    ConfirmedTransactionData,
+                    SignedTransactionData,
+                    WalletData,
+                });
+                container.singleton(
+                    IoC.BindingType.DataTransferObjectService,
+                    Services.AbstractDataTransferObjectService,
+                );
+                container.singleton(IoC.BindingType.AddressService, AddressService);
+                container.singleton(IoC.BindingType.ClientService, ClientService);
+                container.singleton(IoC.BindingType.KeyPairService, KeyPairService);
+                container.constant(IoC.BindingType.LedgerTransportFactory, async () => { });
+                container.singleton(IoC.BindingType.LedgerService, LedgerService);
+                container.singleton(IoC.BindingType.PublicKeyService, PublicKeyService);
+                container.singleton(IoC.BindingType.MultiSignatureService, MultiSignatureService);
+            });
+
+            defaultTransferInput = {
+                data: {
+                    amount: 1,
+                    memo: "foo",
+                    to: identity.address,
+                },
+                gasLimit: 21_000,
+                gasPrice: 5,
+                nonce: "1",
+                signatory: new Signatories.Signatory(
+                    new Signatories.MnemonicSignatory({
+                        address: identity.address,
+                        privateKey: "privateKey",
+                        publicKey: "publicKey",
+                        signingKey: identity.mnemonic,
+                    }),
+                ),
+            };
+        });
+
+        it("should calculate fee for a transfer transaction", async () => {
+            nock.fake(/.+/)
+                .get("/api/node/fees")
+                .reply(200, loader.json(`test/fixtures/client/feesByNode.json`));
+
+            const transaction = await transactionService.transfer(defaultTransferInput);
+            const fee = await feeService.calculate(transaction);
+            assert.equal(fee.toString(), "0");
```

Comment:
can you also add a test that asserts a value different from 0? this does not confidently test that it can calculate a value